client/js/supporters/settings/mailchimp-integration-settings.js

```diff
@@ -14,11 +13,11 @@ function init(modalID$) {
   const pathPrefix = `/nonprofits/${app.nonprofit_id}`
   var state = {
     submitForm$: flyd.stream()
-  , tagMasters$: flyd.map(R.prop('body'), request({method: 'get', path: pathPrefix + '/tag_masters'}).load)
+  , tagMasters$: flyd.map(r => r.body, request({method: 'get', path: pathPrefix + '/tag_masters'}).load)
   }
 
-  const emailLists$ = flyd.map(R.prop('body'), request({method: 'get', path: pathPrefix + '/email_lists'}).load)
-  state.selectedTagMasterIds$ = flyd.map(R.map(ls => ls.tag_master_id), emailLists$)
+  const emailLists$ = flyd.map(r => r.body, request({method: 'get', path: pathPrefix + '/email_lists'}).load)
+  state.selectedTagMasterIds$ = flyd.map(emailLists$.map(ls => ls.tag_master_id))
```

Comment:
This one is wrong, I think.

Reply:
Can you say more about what looks wrong here? As far as I can tell it's the same as other similar places, switching from `R.prop('thing')` to `i => i.thing` and from `R.map(f, thing)` to `thing.map(f)`